srcjs/input_binding_select.js

```diff
@@ -115,7 +115,14 @@ $.extend(selectInputBinding, {
     $(el).trigger('change');
   },
   subscribe: function(el, callback) {
+    var thiz = this;
     $(el).on('change.selectInputBinding', function(event) {
```

Comment:
I think if you use an arrow function for the `change.selectInputBinding` callback then you don't need to explicitly capture `this` (just refer to `this` in arrow function).

Reply:
Thanks, fixed. I thought we couldn't use ES2015 here but you're right, we can.